### DISCUSSION TITLE
How to enable Synology high avaibility with only NVME volumes

### DISCUSSION THREAD

**Discussion by delta-whiplash** [007revad/Synology_enable_Deduplication#84] [Q&A] [answered]
Hello,
I've noticed that you're skilled at bypassing Synology menu restrictions.
I was wondering if you have any scripts that would allow me to use an NVMe volume with Synology High Availability

**Comment by 007revad** [comment, marked as answer]
Your title asked a slightly different question to the body of your message.

To answer the "with **only** NVMe volumes" part of the title: 
Getting DSM to install and run from NVMe drives is complicated, but I do know of someone who has done it. It would be far easier to install a pair of 2.5 inch SATA SSDs for DSM to run off and then having your data on NVMe volumes.

To answer the "allow me to use an NVMe volume with Synology High Availability" question:
https://github.com/007revad/Synology_HDD_db should allow you to create NVMe storage pools and volumes from storage manager

With Synology High Availability you need to run the script on the Active server, then reboot. And once the Active server is working as you want you need to make the Passive server Active and repeat the process.
